src/Routers/UsersRouter.js

```diff
@@ -357,6 +357,30 @@ export class UsersRouter extends ClassesRouter {
     );
   }
 
+  handleResetPassword(req) {
```

Comment:
It looks that the additions in this file are not necessary. You can use `resetPassword` function directly from the GraphQL API.

Reply:
Can you please point at which function you are talking about?? I am unable to find it

Reply:
This one: https://github.com/parse-community/parse-server/blob/master/src/Controllers/UserController.js#L260

Reply:
Hi @davimacedo , I updated both file userControllers.js & [userMutation.js](https://github.com/parse-community/parse-server/blob/e88f2e38f9c38151869055878e52f1da8f7f6f99/src/GraphQL/loaders/usersMutations.js), as suggested. Please let me know if more changes are needed.